pkgs/development/web/postman/default.nix

```diff
@@ -96,10 +127,11 @@ stdenv.mkDerivation rec {
     popd
   '';
 
+
   meta = with lib; {
-    homepage = "https://www.getpostman.com";
+    homepage = "https://www.postman.com/";
     description = "API Development Environment";
-    license = licenses.postman;
+    #license = licenses.postman;
```

Comment:
Why is the license commented now? Does the `postman` license exist in `licenses`? If not, please review whether it is an unfree license and change `license` to `licenses.unfree` or another appropriate license.

Reply:
Sorry, I had commented it to test the build. Uncommenting it yields the below, but none of those are feasible solutions (a) environment variable isn't used with `nix build .#postman`, b) I have already done this in my system flake, but it's not used, because I'm building from this repo's flake which doesn't have that option enabled,  c) irrelevant, because these aren't flake commands.

```bash
$ nix build .#postman
error: Package ‘postman-9.2.0’ in /nix/store/npq55pcd6p5m90k2360q8mvgl93cipzx-source/pkgs/development/web/postman/default.nix:136 has an unfree license (‘postman’), refusing to evaluate.

       a) To temporarily allow unfree packages, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNFREE=1

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowUnfree = true; }
       in configuration.nix to override this.

       Alternatively you can configure a predicate to allow specific packages:
         { nixpkgs.config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [
             "postman"
           ];
         }

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowUnfree = true; }
       to ~/.config/nixpkgs/config.nix.
(use '--show-trace' to show detailed location information)
```

Reply:
Uncommented in 7eb8b7f484d48e3f3559e1d8a218de073fd98a44 .

Reply:
I usually edit `flake.nix`:
```nix
  legacyPackages = forAllSystems (system: import ./. { inherit system; config.allowUnfree = true; });
```

And set `git update-index --assume-unchanged flake.nix`. This way, I don't inadvertently commit the changed flake.nix.

But yes, it is a nuissance.

Reply:
That's a pretty decent solution considering the alternatives. To be clear, `allowUnfree` is always present in your `flake.nix`, right?

Reply:
> Please, squash the fixes into a single commit!

Done in 9608b99aa4e4d78013da5fcfd45d67c6fee8e6b7 . My commit message is multi-line to reflect the changes to `installPhase` and `meta.homepage`. Please let me know if that needs to be stripped out.

Reply:
Exactly, when opening a new working tree. I only touch `flake.nix` when necessary. I don't often need to compile unfree packages.

Reply:
Thanks, I understand. I really appreciate your help with this PR!